packages/neo4j/src/utils.ts

```diff
@@ -16,6 +16,7 @@ export const format = {
     for (const key in object) {
       const value = object[key]
       if (value instanceof Date) newObject[key] = value.toISOString()
+      else if (value instanceof Object) newObject[key] = format.to(value)
```

Comment:
I think this would be better handled where needed in the adapter. just split the stuff that is an object and run formatting on it.

Reply:
Thanks for the suggestion. It's addressed here: https://github.com/nextauthjs/adapters/pull/316/commits/2b2f588af32820eedd2277cf73ac8d23492e116d